shipyard-controller/main_test.go

```diff
@@ -1027,7 +1027,7 @@ func Test__main_SequenceStateParallelStages(t *testing.T) {
 			return false
 		}
 		return true
-	}, 3*time.Second, 100*time.Millisecond)
+	}, 10*time.Second, 100*time.Millisecond)
```

Comment:
technically not related to this PR, but this should improvve the reliability of the component tests which have been flaky lately